build.gradle

```diff
@@ -286,6 +286,14 @@ subprojects {
         }
     }
 
+    task testModules(type: Exec) {
```

Comment:
This is failing for me because:
```
Error occurred during initialization of boot layer
java.lang.module.FindException: Unable to derive module descriptor for /.../micrometer/implementations/micrometer-registry-new-relic/build/libs/micrometer-registry-new-relic-1.10.0-SNAPSHOT.jar
Caused by: java.lang.module.FindException: Automatic-Module-Name: micrometer.registry.new.relic: Invalid module name: 'new' is not a Java identifier
```

Reply:
Alo, this way, this is executed against the bom, the samples and the benchmarks too, I think this also fails since the bom does not have a jar.

I think it is an easy fix, we just need to put a few lines below, after this section:
```
            jar {
                manifest.attributes.put('Automatic-Module-Name', project.name.replace('-', '.'))
                metaInf {
                    from "$rootDir/LICENSE"
                    from "$rootDir/NOTICE"
                }
            }
```

Reply:
I made a separate issue for the new relic automatic module name: https://github.com/micrometer-metrics/micrometer/issues/3412